core/src/main/java/io/questdb/mp/Worker.java

```diff
@@ -152,13 +152,17 @@ public void run() {
     }
 
     private void onError(int i, Throwable e) throws Throwable {
-        if (haltOnError) {
-            throw e;
-        }
-        if (log != null) {
-            log.error().$("unhandled error [job=").$(jobs.get(i).toString()).$(", ex=").$(e).$(']').$();
-        } else {
-            e.printStackTrace();
+        try {
```

Comment:
try... finally is excessive here, throwing out of finally block is not recommended